dogfood/main.tf

```diff
@@ -172,10 +171,16 @@ resource "coder_agent" "dev" {
   startup_script         = <<-EOT
     set -eux -o pipefail
 
+    # change to home
+    cd /home/coder
+
     # install and start code-server
     curl -fsSL https://code-server.dev/install.sh | sh -s -- --method=standalone --prefix=/tmp/code-server --version 4.8.3
     /tmp/code-server/bin/code-server --auth none --port 13337 >/tmp/code-server.log 2>&1 &
 
+    # Install and launch filebrowser
+    curl -fsSL https://raw.githubusercontent.com/filebrowser/get/master/get.sh | bash
+    filebrowser --noauth --root /home/coder --port 8080 >/tmp/filebrowser.log 2>&1 &
```

Comment:
I'm not sure about using up port `8080` here, it's a common port used in development. I believe it conflicts with `develop.sh` too. Maybe next after code-sever, `13338`?